[ 2025-02-20 20:29:49,838 ] root - INFO - Entered the start_data_ingestion method of TrainPipeline class
[ 2025-02-20 20:29:49,840 ] root - INFO - Getting the data from mongodb
[ 2025-02-20 20:29:49,841 ] root - INFO - Entered initiate_data_ingestion method of Data_Ingestion class
[ 2025-02-20 20:29:49,841 ] root - INFO - Exporting data from mongodb
[ 2025-02-20 20:29:51,303 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "message": "Starting topology monitoring"}
[ 2025-02-20 20:29:51,303 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: []>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:51,304 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "message": "Starting server monitoring"}
[ 2025-02-20 20:29:51,305 ] pymongo.connection - DEBUG - {"clientId": {"$oid": "67b7436718c65aa7b42cfa10"}, "message": "Connection pool created", "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017}
[ 2025-02-20 20:29:51,310 ] root - INFO - MongoDB connection successful.
[ 2025-02-20 20:29:51,312 ] pymongo.serverSelection - DEBUG - {"message": "Server selection started", "selector": "Primary()", "operation": "find", "topologyDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None>]>", "clientId": {"$oid": "67b7436718c65aa7b42cfa10"}}
[ 2025-02-20 20:29:51,313 ] pymongo.serverSelection - DEBUG - {"message": "Waiting for suitable server to become available", "selector": "Primary()", "operation": "find", "topologyDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None>]>", "clientId": {"$oid": "67b7436718c65aa7b42cfa10"}, "remainingTimeMS": 30}
[ 2025-02-20 20:29:51,622 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 312.99999984912574, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:51,623 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:52,125 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:52,126 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:52,629 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:52,630 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:53,132 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:53,133 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:53,635 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:53,635 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:54,138 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:54,138 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:54,641 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:54,642 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:55,144 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:55,145 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:55,647 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:55,648 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:56,249 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 94.0000000409782, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:56,249 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:56,752 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:56,753 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:57,255 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:57,256 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:57,758 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:57,759 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:58,261 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:58,261 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:58,764 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:58,765 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:59,267 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:59,267 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:29:59,769 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:29:59,770 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:00,273 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:00,274 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:00,775 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:00,776 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:01,660 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 391.0000000614673, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:01,661 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:02,163 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:02,164 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:02,666 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:02,669 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:03,171 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 15.00000013038516, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:03,172 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:03,673 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:03,674 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:04,176 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:04,177 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:04,678 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:04,679 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:05,182 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:05,183 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:05,684 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:05,685 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:06,273 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 77.9999999795109, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:06,274 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:06,776 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:06,777 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:07,279 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:07,280 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:07,781 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:07,782 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:08,284 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:08,285 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:08,788 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:08,789 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:09,291 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:09,292 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:09,794 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:09,795 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:10,297 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:10,298 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:10,802 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:10,803 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:11,380 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 78.99999991059303, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:11,380 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:11,883 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:11,884 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:12,386 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:12,386 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:12,889 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:12,890 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:13,393 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:13,394 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:13,895 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:13,896 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:14,398 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:14,399 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:14,901 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:14,902 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:15,405 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:15,405 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:15,907 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:15,908 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:16,484 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 77.9999999795109, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:16,485 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:16,987 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:16,988 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:17,490 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:17,490 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:17,992 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:17,994 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:18,496 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:18,497 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:18,999 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:19,000 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:19,502 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:19,504 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:20,006 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:20,007 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:20,511 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:20,512 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:21,102 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 93.00000010989606, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:21,103 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:21,606 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "serverHost": "cluster0.5pwaq.mongodb.net", "serverPort": 27017, "awaited": false, "durationMS": 0.0, "failure": "\"AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')\"", "message": "Server heartbeat failed"}
[ 2025-02-20 20:30:21,606 ] pymongo.topology - DEBUG - {"topologyId": {"$oid": "67b7436718c65aa7b42cfa10"}, "previousDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "newDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "message": "Topology description changed"}
[ 2025-02-20 20:30:21,607 ] pymongo.serverSelection - DEBUG - {"message": "Server selection failed", "selector": "Primary()", "operation": "find", "topologyDescription": "<TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>", "clientId": {"$oid": "67b7436718c65aa7b42cfa10"}, "failure": "\"cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)\""}
[ 2025-02-20 20:30:21,608 ] root - ERROR - Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\data_access\proj1_data.py] at line number [49]: cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[ 2025-02-20 20:30:21,610 ] root - ERROR - Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [35]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\data_access\proj1_data.py] at line number [49]: cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[ 2025-02-20 20:30:21,613 ] root - ERROR - Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [86]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [35]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\data_access\proj1_data.py] at line number [49]: cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[ 2025-02-20 20:30:21,616 ] root - ERROR - Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\pipline\training_pipeline.py] at line number [45]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [86]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [35]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\data_access\proj1_data.py] at line number [49]: cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
[ 2025-02-20 20:30:21,618 ] root - ERROR - Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\pipline\training_pipeline.py] at line number [131]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\pipline\training_pipeline.py] at line number [45]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [86]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\components\data_ingestion.py] at line number [35]: Error occurred in python script: [C:\Users\sanch\OneDrive\Desktop\data\vehicle MLops\src\data_access\proj1_data.py] at line number [49]: cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms), Timeout: 30s, Topology Description: <TopologyDescription id: 67b7436718c65aa7b42cfa10, topology_type: Unknown, servers: [<ServerDescription ('cluster0.5pwaq.mongodb.net', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('cluster0.5pwaq.mongodb.net:27017: [Errno 11001] getaddrinfo failed (configured timeouts: socketTimeoutMS: 20000.0ms, connectTimeoutMS: 20000.0ms)')>]>
